packages/ember-metal/lib/utils.js

```diff
@@ -643,6 +643,7 @@ var needsFinallyFix = (function() {
   var count = 0;
   try {
     try {
+      Ember.K();
```

Comment:
@rwjblue I'm not thrilled about this but I don't know how else to fill the void. Please advise.